src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: medium;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  padding: 0;
+}
+
+.home {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  margin: 10px;
+  margin-left: 50px;
+}
+
+.nav-bar {
+  margin-right: 50px;
```

Comment:
Actually it's enough to set paddings to header instead of setting margins for both child elements